editor/plugins/gpu_particles_2d_editor_plugin.cpp

```diff
@@ -292,11 +292,16 @@ void GPUParticles2DEditorPlugin::_generate_emission_mask() {
 	texdata.resize(w * h * 2 * sizeof(float));
 
 	{
+		Vector2 offset(0., 0.);
```

Comment:
The default value is already `Vector2(0, 0)`, so you can skip the assignment. Also, we generally don't use C++-style parentheses initializers and prefer to use `=` instead where we do need assignment.

```suggestion
		Vector2 offset;
```

Reply:
I fixed it with an amend... and then noticed that github has an easy button to commit your suggestion directly...